app/views/catalog/_mods_search_results_document_fields.html.erb

```diff
@@ -6,13 +6,13 @@
     <% if document.mods_display_name.present? %>
       <li>
         <% name = document.mods_display_name.first %>
-        <%= link_to_mods_name(name['link']) %> <%= name['post_text'] %>
+        <%= link_to(name['link'], search_catalog_path(q: "\"#{name['link']}\"", search_field: 'search_author')) %> <%= name['post_text'] %>
```

Comment:
This could be ported back out to a helper, or we could not delete the one we had previously? But.. 🤷